src/server/pkg/work/work.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package work;
+option go_package = "github.com/pachyderm/pachyderm/src/server/pkg/work";
+
+// import "google/protobuf/any.proto";
+
+enum State {
+  RUNNING = 0;
+  COMPLETE = 1;
+  FAILED = 2;
```

Comment:
Can we use `SUCCESS` and `FAILURE` here to match with our other State enums?

Reply:
Sure.